### DISCUSSION TITLE
Browsers display AR screens differently

### DISCUSSION THREAD

**Discussion by hmduc01** [google/model-viewer#2954] [Q&A]
I still don't know where is the problem when the browsers show the screens differently like this, is there any way to make it show all the features in a browse ? .Thanks
1. Chrome : 
![Chrome](https://user-images.githubusercontent.com/89718203/139804282-a7d5c48c-bfa5-4711-adc4-338587bab9bc.PNG)
r

2. Browser and other browsers :
![Browser](https://user-images.githubusercontent.com/89718203/139804383-2f709df1-6f10-4fc5-9757-c617530e4e0f.PNG)
.

**Comment by elalish** [comment]
The HTML features are web-based and so can't translate to a  separate app like Scene Viewer or Quick Look. Only WebXR mode can show them because this is  the only AR mode that stays in  the  browser, hence it being our  default.

**Reply by ghost** [reply]
Yes it looks like the 2nd set is launching into SceneViewer because WebXR is not available on those browsers. Curios if you are you primarily interested in having the variant picker dropdown available everywhere or is it that the experience is not uniform?
